webapp/channels/src/components/post_view/commented_on/commented_on.tsx

```diff
@@ -12,7 +12,7 @@ import * as Utils from 'utils/utils';
 import {stripMarkdown} from 'utils/markdown';
 
 import CommentedOnFilesMessage from 'components/post_view/commented_on_files_message';
-import UserProfile from '../../user_profile/user_profile';
+import UserProfile from '../../user_profile';
```

Comment:
Just for our future reference, this line is actually the fix for the issue. The connected version of `UserProfile` correctly uses the post author's display name

Reply:
@larkox This is the actual fix